.pre-commit-config.yaml

```diff
@@ -61,13 +61,13 @@ repos:
       - id: black
         args: [ "--config", "./pyproject.toml" ]
   - repo: https://github.com/asottile/blacken-docs
-    rev: 1.15.0
+    rev: 1.16.0
     hooks:
       - id: blacken-docs
         alias: black
         additional_dependencies: [black>=22.10.0]
   - repo: https://github.com/pre-commit/mirrors-mypy
-    rev: 'v1.5.0'
+    rev: 'v1.5.1'
```

Comment:
We may have to solve some pre-commit issue before merging this PR

Reply:
I was able to run locally without any issues, triggering the CI again to see if pre-commit checks pass now

Reply:
This was the error in the pre-commit server:
```build of https://github.com/pre-commit/mirrors-mypy:types-PyYAML,types-attrs,attrs,types-requests,types-python-dateutil,apache-airflow@v1.5.1 for python@python3 exceeds tier max size 250MiB: 262.2MiB```

Reply:
We ended up removing MyPy checks from pre-commit and adding them to the CI, to overcome this issue.